src/vendor/vendor.c

```diff
@@ -62,61 +62,42 @@
 #include <sys/time.h>
 #include <sys/stat.h>
 
-#include <ccsp_message_bus.h>
-#include <ccsp_base_api.h>
-#include <dslh_definitions_database.h>
-
+#include <rbus.h>
+#include <rbus_value.h>
+#include <rbuscore.h>
```

Comment:
I'm getting compilation errors when I try to build this.
I checked out the 2024Q3 release and built for RaspberryPi 4B 32 Bit.
Then I modified `meta-rdk-broadband/recipes-rdkb/usp-pa/usp-pa.bb` to point to the repo and branch used in this pull request:
`SRC_URI += "git://github.com/Ssandhyarani/usp-pa-vendor-rdk;protocol=http;branch=usppa-rbus;rev=8c76b8b53a8081c8d7e6fe05d9ff2cca943b61be;name=usppa;destsuffix=usp-pa-vendor-rdk"`
Then I did
`bitbake -c do_cleanall usp-pa` followed by `bitbake usp-pa`.
The build error I get is:
`| In file included from ../obuspa/src/vendor/vendor.c:67:
| /n/rdkb-2024q3/build-raspberrypi4-rdk-broadband/tmp/work/cortexa7t2hf-neon-vfpv4-rdk-linux-gnueabi/usp-pa/1.0-r0/recipe-sysroot/usr/include/rbus/rbuscore.h:22:10: fatal error: rtMessageHeader.h: No such file or directory
|    22 | #include <rtMessageHeader.h>
|       |          ^~~~~~~~~~~~~~~~~~~
`
I guess either a dependency needs adding to the bb file, or an include path needs adding to vendor.am.
Please check that the PR builds for RPi 32 bit and 64 bit builds.

Reply:
Fixed by adding `-I${STAGING_INCDIR}/rtmessage` to CFLAGS in usp-pa.bb

Reply:
Please submit a PR with the changes to `meta-rdk-broadband/recipes-rdkb/usp-pa` that are linked with this commit.